### ISSUE TITLE
Newlines are stripped from the response

### ISSUE THREAD

**Issue by mieubrisse** [kurtosis-tech/kurtosis-package-indexer#48]
https://kurtosistech.slack.com/archives/C05DZ4UABDZ/p1698675939631249?thread_ts=1698675579.006159&cid=C05DZ4UABDZ

**Comment by tedim52** [comment]
I've determined this to be caused by how the package we used to parse docstrings parses the args/fields of the docstring. 

When parsing the args of a docstring, `docstring` removes newline characters `"\n"` and replaces them with `" "` as documented [here](https://github.com/tilt-dev/starlark-lsp/blob/84c13fe0ff94f5586b2ee454820cf42b9ec583ae/pkg/docstring/docstring.go#L99C1-L104C1).
```
// Field represents single "<name>: blah-blah-blah" definition.
type Field struct {
	Name string // name of the field
	Desc string // field's description, "\n" is replaced with " "
}
```
One option is to fork the package, adjust this specific behavior, and use our forked package (similar to what we do for `stacktrace`) but looking for cheaper ways to fix before doing that.
